pkg/ipamplugin/ipam_plugin.go

```diff
@@ -109,7 +109,8 @@ func Main(version string) {
 
 type ipamArgs struct {
 	cnitypes.CommonArgs
-	IP net.IP `json:"ip,omitempty"`
+	IP             net.IP                        `json:"ip,omitempty"`
+	CALICO_POD_UID cnitypes.UnmarshallableString `json:"podUID,omitempty"`
```

Comment:
Do you have to use the non-standard naming convention here?

Reply:
Yeah, the code that parses this looks for env vars that match the field names in the struct.

I suppose I could either:
- Be consistent with the other env vars (what this PR does currently)
- Be consistent with the other go struct names (making the env var inconsistent)

I don't really have a strong preference either way.